Reportedly/Services/Network/UserStorage/UserManager.swift

```diff
@@ -12,6 +12,24 @@ final class UserManager {
     
     static let shared = UserManager()
     
-    // TODO: - Update to get actual token from sever later on
-    public var token: Token? { nil }
+    public var token: Token? {
+        didSet {
+            if let token = token {
+                UserDefaults.userCredentials = UserCredentials(appToken: token, userId: user?.id ?? -1)
```

Comment:
imo, I think we should validate user.id instead of hard code value `-1`. What do you think? 😆

Reply:
sure, let me update the token structure here a bit